docs/querying/basics.md

```diff
@@ -87,17 +86,17 @@ Examples:
 
 ### Instant vector selectors
 
-Instant vector selectors allow the selection of a set of time series and a
-single sample value for each at a given timestamp (instant): in the simplest
-form, only a metric name is specified. This results in an instant vector
+Instant vector selectors allow the selection of a set of time series (in a time range query) 
+and a single sample value for each at a given timestamp (in an instant query).  In the simplest
+form, only a metric name is specified, which results in an instant vector
```

Comment:
This change doesn't seem right to me.  The original is trying to define "instant vector" as a thing independent of the query it appears in.  "timestamp (instant)" means the word "timestamp" means the same as the word "instant".

Reply:
I guess for some the word instant is overloaded, so we could say `(instant in time)` or just not have it there?

Reply:
Or point in time, which might be more English